src/services/app-versions-service.ts

```diff
@@ -51,3 +51,28 @@ export const defaultVersionByAppId = async (appId: AppId, useLiveVersion = false
 
   return validVersion;
 };
+
+export const getAppVersionById = async (appVersionId: AppVersionId): Promise<AppVersion> => {
+  try {
+    const path = getAppVersionsByAppIdUrl(appVersionId);
+    const url = appsUrlBuilder(path);
+    logger.debug(`fetching logs url: ${url}`);
+    const response = await execute<Version>(
+      {
+        url,
+        headers: { Accept: 'application/json' },
+        method: HttpMethodTypes.GET,
+      },
+      appVersionsSchema,
+    );
+
+    return response.appVersion;
```

Comment:
is it `response.appVersion` ? or `response.appVersions` ? because this is an array

Reply:
Why do you say it's returning an array this is a new endpoint that returns a single app version by id
@maorb-dev